security-considerations.md

```diff
@@ -1,3 +1,6 @@
 ## Security Considerations
 
-This Working Group Note does not introduce any new security considerations.
+This Working Group Note does not introduce any new security considerations. However, when implementing WCAG 2 success criteria in the context of non-web ICT, accessibility features could potentially be exploited to bypass software security features. It is best practice to choose implementations that reduce the potential for unauthorized access (e.g. ability to bypass logins and CAPTCHAs) or introduction of malicious software by cyberattackers. 
```

Comment:
Might it be worth saying something like:

> However, when implementing WCAG 2 success criteria in the context of non-web ICT, accessibility features, as with any additional features, could potentially be exploited to bypass software security features.

This just highlights that it isn't a unique aspect of accessibility features but a challenge with adding _any_ additional features.

Reply:
The latest updates include the WAI proposed text as-is (from the Google doc).